exporter/elasticsearchexporter/internal/serializer/otelserializer/profile_test.go

```diff
@@ -152,3 +153,55 @@ func TestSerializeProfile(t *testing.T) {
 func isWithinLastSecond(t time.Time) bool {
 	return time.Since(t) < time.Second
 }
+
+func BenchmarkSerializeProfile(b *testing.B) {
+	ser, err := New()
+	require.NoError(b, err)
+
+	profiles := pprofile.NewProfiles()
```

Comment:
Can you use the pprofiletest to create profiles in a more abstract and readable way (example: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/16ac39d8db602bd6207685c90c61739354038e48/pkg/pdatatest/pprofiletest/profiles_test.go#L19)

Also, it should mostly avoid rewriting this code when/if internal data layout changes.